dashboard/app/views/levels/_external.html.haml

```diff
@@ -33,10 +34,12 @@
   .clear
   .video-container.external-video-container{{style: 'max-width: 970px; margin: auto'}}
   .button-box{{style: 'width: 970px; margin: auto'}}
-    %a.btn.btn-large.btn-primary.next-lesson.submitButton.pull-right= t('continue')
     - if video.download
       %a.video-download.btn.btn-large.pull-left{href: video.download}
         = t('video.download')
+    .buttons.below-content-right
```

Comment:
I switched the order of the download and continue buttons so that if the user tab-navigates, they go from left (download button) to right (continue button).